Microsoft.DotNet.DarcLib.Tests/RemoteTest.cs

```diff
@@ -0,0 +1,97 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Moq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.DarcLib.Tests
+{
+    public class RemoteTest
+    {
+        [Fact]
+        public async Task MergeDependencyPullRequest()
```

Comment:
[nit] It's good practice for the test name to indicate what you're actually testing, not just the method you're exercising.

Reply:
In this case you're validating the commit description, while the name implies that you're testing the merging of dependency update pull requests.